db/migrations/20180430094742_add_name_to_movies.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function (knex) {
+  return knex.schema.table('movies', (table) => {
```

Comment:
nit: this can be collapsed into a single line `knex.schema.table('movies', (table) => table.text('name'))` since you are adding just a single column